### PR TITLE
Add circle.yml

### PR DESCRIPTION
**Pull request by S2606**
Added circle.yml file for CircleCI Integration.

Closes https://github.com/coding-blocks/Chatter/issues/11

### CODE REVIEW

**Comment by the-dagger** [comment]
@S2606 Please test the CircleCI integration on your fork first before sending a PR.

**Comment by S2606** [comment]
@the-dagger Could you please tell me how to test on my fork??

**Comment by S2606** [comment]
@the-dagger I have tested for my fork and its working well.

**Comment by championswimmer** [comment]
add the outputs/ directory to build artifacts please 
check gtihub.com/coding-blocks/DigitalOceanApp for a reference config

**Comment by S2606** [comment]
@championswimmer Made the required changes. Results below

https://circleci.com/gh/S2606/Chatter/3

**Comment by championswimmer** [comment]
why are we doing a connectedCheck ? Do we even have UI tests on the project ?

**Comment by S2606** [comment]
@championswimmer Pls review it

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        - /usr/local/android-sdk-linux/tools
+        - /usr/local/android-sdk-linux/build-tools/25.0.3
+test:
+  pre:
+    - emulator -avd circleci-android22 -no-window:
+        background: true
+        parallel: true
+  override:
+    - ./gradlew clean assembleDebug
+    - circle-android wait-for-boot:
```
Comment (championswimmer) [inline circle.yml]: why wait for boot when no connectedCheck ? 

Like please can you read your code before you submit.
Do not care to submit code that you have no idea about.

**Comment by S2606** [comment]
I am sorry @championswimmer for my mistakes.

**Comment by abishekvashok** [comment]
@championswimmer next time pls squash and merge :)

**Comment by abishekvashok** [comment]
@championswimmer please enable it :)

**Pull request merged**